community/ndp/kernelextension/src/main/java/org/neo4j/ext/NDPKernelExtension.java

```diff
@@ -94,7 +94,8 @@ public Lifecycle newKernelExtension( Dependencies dependencies ) throws Throwabl
         final Config config = dependencies.config();
         final GraphDatabaseService gdb = dependencies.db();
         final GraphDatabaseAPI api = (GraphDatabaseAPI) gdb;
-        final Log log = dependencies.logService().getInternalLog( Sessions.class );
+        final LogService logging = dependencies.logService();
+        final Log log = logging.getInternalLog( Sessions.class );
```

Comment:
@cleishm there is just one store right now though :) In my mind, the internal log is for debugging the dbms by our engineers, not per-store information. 

But there's a bigger architectural disagreement here on where multi-tenancy lives. Suggest we find a whiteboard this summer to talk through that.